### ISSUE TITLE
Recipe App kanban setup

### ISSUE THREAD

**Issue by sevinchek** [the-catalystmc/recipe-app#7]
- This project has two contributors: @sevinchek @the-catalystmc
- The link of our github project is [https://github.com/the-catalystmc/recipe-app/projects/1](https://github.com/the-catalystmc/recipe-app/projects/1)

**Comment by Lameck1** [comment]
Hi, @sevinchek & @the-catalystmc, 

You are doing well so far. We have just had a few issues to look into before we can move ahead.

## Required Changes ♻️
- [ ] This first task on the board, **_`[4pt] (Group) Layout wireframes`_**, is too general. Looking further into it, it's not really a task but a general instruction to keep in mind while developing our user interfaces. Tasks should written in this format:
       **Title** => **`Do this!`**
       **Description** => **`What to do` OR `How to do it!`**
       Therefore, tasks should be describing what **_`features`_** of the app to build in the **`Titles`** and how to implement those features in the **`Descriptions`**
       Other tasks too are too general, consider breaking them into smaller features that can be built as one.

### Requirements that are missing corresponding tasks that should be added

1. Food list:
    * Should display a list of food as in the wireframe.
    * Should lead to the form that allows users to add new food.
2. Recipe details:
    * If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
    * If the user is the owner of the recipe, should lead to the form that allows user to add new food.
3. Make sure there are no N+1 queries happening.
4. Create navigation menu that allows users to open all of the pages you created.

## One guiding example you can use (Requirement vs Task and Task Description)

- Requirement
   - Login page and registration page:
      - Should be built with Devise.

- Task and Task Description
   You can divide the requirement into several tasks to create achievable milestones i.e.
     * [1pt] Add and install the devise gem
     * [1pt] Setup the devise user model
     * [1pt] Add Signup and Login links     

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not create another issue re-reviews. You should use this same issue submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
